python/ray/serve/_private/deployment_method_executor_node.py

```diff
@@ -74,6 +74,10 @@ def to_json(self) -> Dict[str, Any]:
             "uuid": self.get_stable_uuid(),
         }
 
+    def get_return_type(self) -> str:
```

Comment:
Can we add a docstring mentioning that the types are calculated using `type_to_string` from `gradio_utils.py`?